account_early_payment_discount/README.rst

```diff
@@ -0,0 +1,63 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================================
+Payment Term - Early Payment Discount
+=====================================
+
+This module adds **Early Payment Discount** information in Payment Term model.
+
+Look at ``account_operation_rule_early_payment_discount`` module in https://github.com/OCA/bank-statement-reconcile
```

Comment:
The module will be called 'account_reconcile_rule_early_payment_discount' (to be confirmed by @fclementic2c ) but its repo is now OCA/account-reconcile.